src/graph/graphcontext_retrieve.c

```
@@ -0,0 +1,236 @@
+#define LOAD_API(name)                                                    \
+	if (name == NULL) {                                                   \
+		name = RedisModule_GetSharedAPI (ctx, #name) ;                    \
+		if (name == NULL)                                                 \
+		RedisModule_Log (ctx, "warning", "missing FalkorDB API: %s",      \
+					#name) ;                                              \
+	}
+
+	if (graph_load != NULL && GraphStubType_Get != NULL) {
+		return true ;
+	}
+
```

Cppcheck (2.20.0)

[error] Memory leak

(memleak)